plugins/inputs/postgresql/postgresql.go

```diff
@@ -42,6 +40,20 @@ var sampleConfig = `
   ##
   address = "host=localhost user=postgres sslmode=disable"
 
+  ## connection configuration.
+
+  ## maxidle - specify the number of idle connections to maintain. see sql.DB for details.
+  ## default is 1. setting to 0 to force disconnect after use.
+  max_idle = 1
```

Comment:
> flat out incorrect.

Yes, because of other problems (such as setting `max_conns` > 1 being pointless). As soon as they are fixed, as long as `max_idle` < `max_conns`  the statement is completely accurate.

There is however one use case I didn't think of. Setting to `0` would allow restoring the previous behavior of tearing down the connection after every polling interval. Though care would need to be taken to ensure it's not torn down after every query.